### ISSUE TITLE
Storybook run command terminates right after it successfully ran message

### ISSUE THREAD

**Issue by NechiK** [nrwl/nx#26087]
### Current Behavior

After updating to Storybook 8, when I run the command `npx nx storybook angular-test-storybook`, it builds stories and opens the browser, so I can see that modules are building, BUT it terminates right after the message `Successfully ran target storybook for project angular-test-storybook`. It worked when I initially created the NX workspace and used Storybook 7.

### Expected Behavior

 The command `npx nx storybook angular-test-storybook` should start watch mode as it was with Storybook 7.

P.S. I've tested Storybook 8 on a clean Angular environment (without NX), and it works properly.

### GitHub Repo

https://github.com/NechiK/nx-angular-storybook-8-issue

### Steps to Reproduce

1. Create Nx workspace `npx create-nx-workspace@latest angular-test-storybook --preset=angular-monorepo`
2. Run `npx nx add @nx/storybook`
3. Add configuration `npx nx g @nx/angular:storybook-configuration angular-test-storybook`
4. Run storybook `npx nx storybook angular-test-storybook`. It runs successfully, and I can view it in the browser.
5. Upgrade to Storybook 8 `npx storybook@latest upgrade --config-dir apps/angular-test-storybook/.storybook`
6. Re-run the storybook: `npx nx storybook angular-test-storybook`. It runs a build, shows the Storybook version message with URLs, and `Successfully ran target storybook for project angular-test-storybook`. And immediately terminates.

### Nx Report

```shell
NX   Report complete - copy this into the issue template

Node   : 18.20.2
OS     : darwin-x64
npm    : 10.5.0

nx                 : 19.1.0
@nx/js             : 19.1.0
@nx/jest           : 19.1.0
@nx/linter         : 19.1.0
@nx/eslint         : 19.1.0
@nx/workspace      : 19.1.0
@nx/angular        : 19.1.0
@nx/cypress        : 19.1.0
@nx/devkit         : 19.1.0
@nx/eslint-plugin  : 19.1.0
@nx/storybook      : 19.1.0
@nrwl/tao          : 19.1.0
@nx/web            : 19.1.0
@nx/webpack        : 19.1.0
typescript         : 5.4.5
---------------------------------------
Registered Plugins:
@nx/storybook/plugin
---------------------------------------
Community plugins:
@storybook/angular : 8.1.3
```

### Failure Logs

```shell
No failure logs
```

### Operating System

- [X] macOS
- [ ] Linux
- [ ] Windows
- [ ] Other (Please specify)

### Additional Information

I had this issue when I used Nx 19.0.6 and 19.0.8 versions too.

**Comment by NechiK** [comment]
I had the same issue when I used Nx 19.0.6 and 19.0.8

**Comment by ryandegruyter** [comment]
I had the same issue, check this out: https://github.com/storybookjs/storybook/issues/27355

**Comment by rhutchison** [comment]
Thanks @ryandegruyter 

Resolved by https://github.com/storybookjs/storybook/pull/27361

https://github.com/storybookjs/storybook/issues/27355#issuecomment-2129671697

**Comment by NechiK** [comment]
Thanks @ryandegruyter 

They have fixed it in 8.1.4 release today.
